Expensify/App
# scale image by maximum of width or height scale factor

PBing: @tylerkaraszewski 

cc @thienlnam

This calculates the scale factor of both the height and width of an image and scales the image by the maximum of the two.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/146484

### Tests
1) Post an image like this one where the height is greater than the width:

![image](https://user-images.githubusercontent.com/47436092/99743068-bcb81980-2a89-11eb-9da0-42d9f0bcf917.png)

2) Post an image like this one where the width is greater than the height:

![image](https://user-images.githubusercontent.com/47436092/99743166-f38e2f80-2a89-11eb-93fb-5e9eef27e64b.png)

3) Click on the image to enlarge it, and ensure that it taking up as much space as possible and is not cropped on any side.

4) Repeat step 3 on all platforms

5) Cry out in despair after you gather screenshots on web, desktop, and iOS, only to have the Android app crash your whole computer, erasing the work done on this PR so far.

6) Start over, re-write the PR, gathering screenshots on all platforms. Maybe we'll skip android this time (I promise the crash was unrelated)....

### Screenshots

Web:

![image](https://user-images.githubusercontent.com/47436092/99743122-d9545180-2a89-11eb-80fd-ebb2eea29a88.png)

![image](https://user-images.githubusercontent.com/47436092/99743177-fee15b00-2a89-11eb-92f8-4d82b872ad45.png)

Desktop:

![image](https://user-images.githubusercontent.com/47436092/99743589-d443d200-2a8a-11eb-9f06-d9ab6e9e0562.png)

![image](https://user-images.githubusercontent.com/47436092/99743617-e756a200-2a8a-11eb-8dde-00953bc7ad4b.png)

iOS:

![image](https://user-images.githubusercontent.com/47436092/99743938-93988880-2a8b-11eb-913b-70e752cb679f.png)

![image](https://user-images.githubusercontent.com/47436092/99744004-b32fb100-2a8b-11eb-9d50-ebbe9b0db062.png)
